src/cephadm/cephadm

```diff
@@ -7894,8 +7939,26 @@ def _get_parser():
         default=['cephadm'],
         help='packages')
 
+    registry_login_help = """Give cephadm login information for an authenticated registry (url, username and password).
+Cephadm will attempt to log the calling host into that registry::
+
+      cephadm registry-login --registry-url [REGISTRY_URL] --registry-username [USERNAME]
+                             --registry-password [PASSWORD]
+
+Can also use a JSON file containing the login info formatted as::
+
+      {
+       "url":"REGISTRY_URL",
+       "username":"REGISTRY_USERNAME",
+       "password":"REGISTRY_PASSWORD"
+      }
+
+and turn it in with command::
+
+      cephadm registry-login --registry-json [JSON FILE]"""
```

Comment:
also really verbose for a man page? maybe this would be better in the regular cephadm documenation?

Reply:
We could do something like 

```python
if 'conf' in sys.modules:
  import conf
  verbose_help = 'man' not in conf.tags
```
But I don't know if it is worth it.